app/__init__.py

```diff
@@ -30,24 +27,15 @@
 
 import stripe
 from app.settings import get_settings
-from app.helpers.flask_ext.helpers import SilentUndefined, camel_case, slugify, MiniJSONEncoder
-from app.helpers.payment import forex
 from app.models import db
-from app.models.user import User
-from app.models.event import Event
-from app.models.session import Session
 from app.api.helpers.jwt import jwt_authenticate, jwt_identity
-from helpers.formatter import operation_name
-from app.helpers.data_getter import DataGetter
-from app.helpers.flask_ext.errors import NotFoundError, PermissionDeniedError, ServerError, ValidationError
-from apscheduler.schedulers.background import BackgroundScheduler
-from app.helpers.data import DataManager, delete_from_db
-from app.helpers.helpers import send_after_event
-from app.helpers.cache import cache
-from helpers.helpers import send_email_for_expired_orders
+from app.api.helpers.cache import cache
 from werkzeug.contrib.profiler import ProfilerMiddleware
 from app.views import BlueprintsManager
-from app.helpers.auth import AuthManager
+from app.api.helpers.auth import AuthManager
+from app.models.event import Event, EventsUsers
```

Comment:
Actually, they are required to build relationships. I'm not exactly clear about this but removing this caused unnecessary import cycle but yeah these all were there before this refactoring PR